code/controllers/Processes/air.dm

```diff
@@ -1,22 +1,121 @@
-/datum/controller/process/air/setup()
+var/kill_air = 0
+
+var/global/datum/controller/process/air_system/air_master
+
+/datum/controller/process/air_system
+	var/list/excited_groups = list()
+	var/list/active_turfs = list()
+	var/list/hotspots = list()
+
+	//Special functions lists
+	var/list/turf/simulated/active_super_conductivity = list()
+	var/list/turf/simulated/high_pressure_delta = list()
+
+	var/current_cycle = 0
+	var/failed_ticks = 0
```

Comment:
Is this even necessary anymore? It's currently utilized by the MC (along with air_processing_killed), but I"m not sure if it's really necessary anymore or not. Meh.